multiversx_sdk_cli/cli_contracts.py

```diff
@@ -28,14 +30,16 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
 
     sub = cli_shared.add_command_subparser(subparsers, "contract", "new",
                                            "Create a new Smart Contract project based on a template.")
-    sub.add_argument("name")
+    sub.add_argument("--name", help="The name of the contract. If missing, the name of the template will be used.")
     sub.add_argument("--template", required=True, help="the template to use")
-    sub.add_argument("--directory", type=str, default=os.getcwd(),
-                     help="🗀 the parent directory of the project (default: current directory)")
+    sub.add_argument("--tag", help="the framework version on which the contract should be created")
+    sub.add_argument("--path", type=str, default=os.getcwd(),
+                     help="the parent directory of the project (default: current directory)")
     sub.set_defaults(func=create)
 
     sub = cli_shared.add_command_subparser(subparsers, "contract", "templates",
                                            "List the available Smart Contract templates.")
+    sub.add_argument("--tag", help="the sc-meta framework version reffered to")
```

Comment:
Confirm with Andrei M. that the description is correct. Or it should be `the MultiversX Rust framework version to be used ...`?